webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/smartling/SmartlingPluralFix.java

```diff
@@ -25,6 +26,19 @@ private SmartlingPluralFix() {
                     tu.setPluralForm("many");
                     tu.setPluralFormOther(textUnit.getPluralFormOther());
                     tu.setTarget(textUnit.getTarget());
+
+                    if (!Strings.isNullOrEmpty(textUnit.getRepositoryName())){
```

Comment:
why do you need to test for null or empty for those 3?

Reply:
I just follow this pattern with strings that can be empty or null, but also there's no point in doing the assignation if is either empty or null, right?

Reply:
everywhere else for pojo mapping we don't do that, so let's keep it consistent with other places. Side notes those (repository name, locale, path)shouldn't be empty because it doesn't make sense though code wise there are little checks.

Reply:
Updated